qadi_simple_multi.py

```
@@ -23,62 +23,138 @@
 
 sys.path.insert(0, str(Path(__file__).parent / 'src'))
 
-async def run_qadi_phase(phase_name: str, prompt: str, previous_insights: str = ""):
-    """Run a single QADI phase using simple LLM call."""
+async def run_qadi_phase(phase_name: str, prompt: str, previous_insights: str = "", concrete_mode: bool = False, classification_result=None):
```

Ruff (0.12.2)

Missing return type annotation for public function `run_qadi_phase`

(ANN201)